src/sentry/static/sentry/app/components/header/index.jsx

```diff
@@ -7,9 +10,31 @@ import Broadcasts from './broadcasts';
 import StatusPage from './statuspage';
 import UserNav from './userNav';
 import OrganizationSelector from './organizationSelector';
+import TodoList from '../todos';
 
 const Header = React.createClass({
-  mixins: [OrganizationState],
+  mixins: [ApiMixin, OrganizationState],
+
+  getInitialState: function() {
+    if (location.hash == '#welcome') {
+      return {showTodos: true};
+    } else {
+      return {showTodos: false};
+    }
+  },
+
+  componentDidMount() {
+    $(window).on('hashchange', () => {
```

Comment:
added componentDidUnmount. Didn't see how to do this within react-router, hashHistory didn't seem like what was needed?